cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/templates/etl/realtime/ETLWorkerTest.java

```diff
@@ -64,7 +64,7 @@ public void testSimpleConfig() throws Exception {
     }
 
     WorkerManager workerManager = templateManager.startWorker(ETLWorker.class.getSimpleName(), workerArgs);
-    TimeUnit.SECONDS.sleep(5);
+    TimeUnit.SECONDS.sleep(2);
```

Comment:
This was added since we wanted a very preliminary check on the adapter execution logic in the worker driver. Once we have proper sinks, we will remove this test and have proper tests.